api/src/main/java/com/google/sps/Application.java

```diff
@@ -10,6 +10,18 @@
 
 public class Application {
   public static void main(String[] args) {
+    try {
+      Firebase.init();
+      GmailService.init();
+    } catch (Exception e) {
+      
+    }
```

Comment:
Nothing is happening in this catch block.

Reply:
jeremy and i went through this earlier, he suggested the propagation similar to in your PR, however since this is the main method there's nothing to throw, so I figured the [default exception handler](https://stackoverflow.com/questions/27664104/how-default-exception-handler-work) would handle the caught exception. Am i understanding that correctly?